uint32 timeout
---
int32 result
